File: CardViewObjC/CardViewObjC/ViewController.m
//
//  ViewController.m
//  CardViewObjC
//
//  Created by Phineas.Huang on 2020/6/19.
//  Copyright © 2020 Phineas. All rights reserved.
//

#import "ViewController.h"

#import "BottomSheetViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    [self addBottomSheetView];
}

- (void)addBottomSheetView {
    BottomSheetViewController *bottomSheetVc = [BottomSheetViewController new];
    [self addChildViewController:bottomSheetVc];
    [self.view addSubview:bottomSheetVc.view];
    [bottomSheetVc didMoveToParentViewController:self];

    const CGFloat y = self.view.frame.origin.y;
    const CGFloat height = self.view.frame.size.height;
    const CGFloat width = self.view.frame.size.width;
    bottomSheetVc.view.frame = CGRectMake( 0, y + height, width, height);
}


@end

File: CardViewSwift/CardViewSwift/ViewController.swift
//
//  ViewController.swift
//  CardViewSwift
//
//  Created by Phineas.Huang on 2020/6/19.
//  Copyright © 2020 Phineas. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        addBottomSheetView()
    }
    
    func addBottomSheetView() {
        let bottomSheetVC = BottomSheetViewController()

        self.addChild(bottomSheetVC)
        self.view.addSubview(bottomSheetVC.view)
        bottomSheetVC.didMove(toParent: self)

        let height = view.frame.height
        let width  = view.frame.width
        bottomSheetVC.view.frame = CGRect(x: 0, y: view.frame.maxY, width: width, height: height)
    }


}

